### ISSUE TITLE
Copper Balance

### ISSUE THREAD

**Issue by Kamatsu92** [Stereowalker/C.O.M.B.A.T#43]
Are the Copper Breastplate and leggings intended to provide same level of protection as iron?  If so, can I inquire into the rationale or is this an oversight?  The Helmet and Boots are weaker and coincide with copper additions from SimpleOres Mod.   Is there a way to blacklist Copper from C.O.M.B.A.T on the client side where I play single player only?  If not, are you considering changing the relative strength of the chest plate and leggings? I streamlined OreGen within my game except for Copper which is prevalent in a lot of mods.  I would not give up any of the numerous great features of this mod but I dislike the ore duplication.  This is my first posting to your site so I thank you for your consideration.

**Comment by Stereowalker** [comment]
To be honest, making them the same level was a mistake in the beginning, but I've been riding that train of thought for a while. I left it like that because I figured I want copper to be of a similar level to iron, but a little bit weaker with the exception being its durability. As for being able to blacklist copper on the client, I'll consider it

**Comment by Kamatsu92** [comment]
Thanks, I recently realized there was the difference in durability so that helps...it is just that copper is so prevalent and the ability to make all the "good" armor I want in the beginning without much effort seems wrong somehow. I appreciate your consideration....your mods are great!
v/r
Jerry
    On Tuesday, April 28, 2020, 12:21:46 PM GMT+2, Stereowalker <notifications@github.com> wrote:  
 
 


To be honest, making them the same level was a mistake in the beginning, but I've been riding that train of thought for a while. I left it like that because I figured I want copper to be of a similar level to iron, but a little bit weaker with the exception being its durability. As for being able to blacklist copper on the client, I'll consider it
